index.html

```diff
@@ -1,23 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
 
-  <head>
+<head>
     <meta charset="utf-8">
     <meta name="description" content="Your description goes here">
     <meta name="keywords" content="one, two, three">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,600,800" rel="stylesheet">
 
-    <title>Blank Template</title>
+    <title>Relaxr Landing Page</title>
 
-    <!-- external CSS link -->
+  <!-- external CSS link -->
     <link rel="stylesheet" href="css/normalize.css">
     <link rel="stylesheet" href="css/style.css">
 
-  </head>
+</head>
 
-  <body>
-    <div class="container">
-      <!-- Replace me with your html -->
+<body>
+  <div class= "container">  
+   <header>
+      <h1>Relaxr</h1>
+      <h2>Get piece of mind with a single tap</h2>
+        <div class="buttoncenter">
+          <button class="firstbutton">Get it Now</button>
```

Comment:
When creating classes that are two words or more, try using what's known as kebab case or came case.

- kebab case would be: `this-thing-is-multiple-words` (just put dashes between each word but keep everything lower case)
- camel case would be: `thisThingIsMultipleWords` (capitalize the first letter of each word except for the first)